### DISCUSSION TITLE
How to speed up simulation

### DISCUSSION THREAD

**Discussion by FGarelli** [OpenRadioss/OpenRadioss#1561] [Q&A] [answered]
Hi, I'm trying to perform the following simulation and i would like to know if there are any advices to follow to speed up it.
The model doesn't have many elements so it should be not take that much (I'm talking about more then 7/8 hours for a simulation)
If you have any advice please let me know, i think I'm using wrong type of elements maybe.
[high_def_0000.zip](https://github.com/OpenRadioss/OpenRadioss/files/11689712/high_def_0000.zip)

**Comment by RevolT87** [comment]
Hi FGalerelli,
You may want to use added mass in order to speed-up your computation. In DYNA format, *CONTROL_TIMESTEP keyword and the flag DT2MS may be used in order to setup the added mass control in your model. 
I hope it is helpful.
Regards,
Mathis

**Comment by MarianBulla** [comment, marked as answer]
Hi FGalerelli,
yes, I agree with Mathis, you can increase the computation time dramatically by enabeling mass scalling.
The small timestep is cause by the very small elements in your model:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/1c1916e9-8221-4b43-8210-193fed99ae70)
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/21a50187-40ea-4888-9b45-df9172af80fa)

Letting OpenRadioss increase a little bit the mass of that elements, will speed-up your simulation time.

There are several possibilities how to do it, up to advanced mass scalling (= selective mass scalling).
But you can start by adding this line into the engine file (*_0001.rad file):

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/934cab2b-7e02-43ca-a223-190302873200)

For more details, please review the manual page.

Hope it helps you ?

Thanks and best regards,
Marian

**Reply by FGarelli** [reply]
Thank you, it works!
